src/library/searchquery.cpp

```diff
@@ -166,6 +173,39 @@ QString TextFilterNode::toSql() const {
     return concatSqlClauses(searchClauses, "OR");
 }
 
+CrateFilterNode::CrateFilterNode(const QString& argument,
+                                 TrackCollection* pTrackCollection)
+    : m_argument(argument),
+      m_pTrackCollection(pTrackCollection) {
+    init();
+}
+
+void CrateFilterNode::init() {
+    FieldEscaper escaper(m_pTrackCollection->database());
+    QString escapedArgument = escaper.escapeString(kSqlLikeMatchAll + m_argument + kSqlLikeMatchAll);
```

Comment:
Don't escape. When binding parameters to queries you almost never need to escape manually. Everything in queryutil.h is legacy code that shouldn't be used any more. I didn't remember that I eliminated it from the refactored CrateStorage ;)